lib/buffer.js

```diff
@@ -1272,25 +1272,29 @@ function toFloat(x) {
 
 
 Buffer.prototype.readDoubleBE = function(offset, noAssert) {
+  offset = offset >>> 0;
```

Comment:
hmm... I'd almost prefer to make this an error rather than coerce. If someone just happens to pass in 4294967296 as the offset, they're going to get unexpected results otherwise.

Reply:
I prefer that as well but all other functions right now coerce the numbers to uint32. I will open a separate PR to change that for all functions.